Add <small> example

Elchi3 opened the pull request:
Example for https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small

Just a small example! :laughing: 

Fixes #787

wbamberg commented:
Looks great! Thanks @Elchi3 . I wonder why you apply a custom style in this case and not the others? I don't think this is bad necessarily, just wondering why.

Also I wondered if it might look good to have an `<hr>` between the paragraphs, to emphasise the separation.

Elchi3 commented:
Thanks for the review @wbamberg. I added the custom style to emphasize the separation some more, it wasn't small enough for me to really see a difference between the normal `<p>` and `<small>`.

I've now added the `<hr>` between the paragraphs which is an excellent suggestion! I still kept the smaller `font-size` for `<small>`, because I think it looks better, but with the `<hr>` it might not be needed anymore. Either way would be fine with me.
